libraries/chain/dbs_witness_vote.cpp

```diff
@@ -0,0 +1,44 @@
+#include <scorum/chain/dbs_witness_vote.hpp>
+
+#include <scorum/chain/database.hpp>
+#include <scorum/chain/witness_objects.hpp>
+
+namespace scorum {
+namespace chain {
+
+dbs_witness_vote::dbs_witness_vote(database& db)
+    : _base_type(db)
+{
+}
+
+void dbs_witness_vote::create(witness_id_type witness_id, account_id_type vouter_id)
```

Comment:
Would be better, if 'create' method will return object. There must be uniformity.